ods_ci/tasks/Resources/RHODS_OLM/pre-tasks/oc_is_operator_installed.robot

```diff
@@ -3,6 +3,7 @@ Is RHODS Installed
   IF  "${cluster_type}" == "selfmanaged"
       ${result}=  Run Keyword And Return Status
       ...  Run Keywords
+      ...  Oc Get  kind=OdhDashboardConfig  namespace=redhat-ods-applications  AND
```

Comment:
probably, more reliable ways would be:
- check the operator is installed (i.e., subscription object or catalog source)
- check there are any pods in one of the namespaces